tests/pems/missing_public_key_ecdsa_cert.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Not blocking but it would be nice to have these cert generators scripted and reproducible. I started one in [`pems/sni`](https://github.com/aws/s2n-tls/blob/main/tests/pems/sni/generate.sh) but it would be good to add one to the root.

Reply:
That'd be pretty difficult for this one. I had to manually modify it to remove the last entry. If we scripted it, the script would need to parse the base64, parse the ASN.1, remove the last field of the ASN.1, update the overall length, and then re-encode in base64.

Reply:
hopefully anyone in the future will find this comment if they need to reconstruct it, I guess 😄